runtime/assembly/models/option.ts

```diff
@@ -28,4 +27,11 @@ export class Option<T> {
         return this.value;
     }
     
+    /**
+     * Returns true/false depending on the Option whether it is Some or None
+     * @param bytes - SCALE Encoded bytes
+     */
+    static isArraySomething(bytes: u8[]): bool {
```

Comment:
I think that you can safely remove the `isSet` from the `utils` and use this function in the rest of the places that we used to work with `isSet`